ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -40,6 +40,12 @@ export interface CanvasCourse extends CanvasCourseBase {
   course_code: string
 }
 
+export interface CanvasCourseSection {
+  id: number
+  name: string
+  total_students: number
+}
```

Comment:
I  think since My PR https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/154/files is also needs sections object, this interface need to extend the section interface

How about this?
```
export interface CanvasCourseSection {
 id: number
 name: string
}

export interface CanvasCourseSectionWithStudents extends CanvasCourseSection {
  total_students: number
}
```
If you agree that I will make changes to my PR

Reply:
No, I think maybe we should make `total_students` optional, then both functions can use the same type.